content/docs/authorization/v2.x/configuration/powerflex/_index.md

```diff
@@ -92,26 +92,26 @@ Given a setup where Kubernetes, a storage system, and the CSM for Authorization
 
     - Update the `SKIP_CERTIFICATE_VALIDATION` environment value to `true` or `false` depending on if you want to disable or enable certificate validation of the CSM Authorization Proxy Server.
 
-    - Do not update the `configVersion`. You will notice in the example that it is set to v1.11.0, this ensures that checks on version support do not prevent deployment of the v2.0.0-alpha authorization tech preview.
+    - Do not update the `configVersion`. You will notice in the example that it is set to v1.12.0, this ensures that checks on version support do not prevent deployment of the v2.0.0 version of authorization.
```

Comment:
What checks are we referring to here?

Reply:
This is an Operator check on version.

Reply:
I see this value is not global, there is a configVersion attribute for each module. So should this value be 1.12 for both Auth v1 and v2?

Reply:
Yes, that's correct. This configVersion really has no impact on how authorization is deployed since it's for the sidecar. We're only concerned with the user using the v2.0.0 image when enabling authorization.

Reply:
Ok, we can leave it for now.